app/utils/filter_option_utils.py

```diff
@@ -6,7 +6,75 @@
     Country
 from app.utils.notifications_sender import send_slack_message
 
-
+static_filter_options = {
+    "age": [
+        "Adults (18-64 years)",
+        "Children and Youth (0-17 years)",
+        "Seniors (65+ years)"
+    ],
+    "estimate_grade": [
+        "National",
+        "Regional",
+        "Local",
+        "Sublocal"
+    ],
+    "overall_risk_of_bias": [
+        "Low",
+        "Moderate",
+        "High",
+        "Unclear"
+    ],
+    "population_group": [
+        "Blood donors",
+        "Household and community samples",
+        "Residual sera",
+        "Assisted living and long-term care facilities",
+        "Students",
+        "Contacts of COVID patients",
+        "Non-essential workers and unemployed persons",
+        "Essential non-healthcare workers",
+        "Hospital visitors",
+        "Persons who are incarcerated",
+        "Persons living in slums",
+        "Pregnant or parturient women",
+        "Patients seeking care for non-COVID-19 reasons",
+        "Household and community samples",
+        "Health care workers and caregivers",
+    ],
+    "sex": [
+        "Female",
+        "Male",
+        "Other"
+    ],
+    "source_type": [
+        "Journal Article (Peer-Reviewed)",
+        "Preprint",
+        "Institutional Report",
+        "News and Media"
+    ],
+    "specimen_type": [
+        "Whole Blood",
+        "Dried Blood",
+        "Plasma",
+        "Serum",
+    ],
+    "test_type": [
+        "Neutralization",
+        "CLIA",
+        "ELISA",
+        "LFIA",
+        "Other"
+    ],
+    "gbd_region": [
+        'Central Europe, Eastern Europe, and Central Asia',
+        'High-income',
+        'Latin America and Caribbean',
+        'North Africa and Middle East',
+        'South Asia',
+        'Southeast Asia, East Asia, and Oceania',
+        'Sub-Saharan Africa'
+    ]
+}
 def get_filter_static_options():
```

Comment:
This function is copying the code from the above dict directly. Seems like this function isn't really needed, and we can just get the keys from static_filter_options to come up with all_filter_types. If we need this function for some reason because it is used in another file, then have it return static_filter_options. In that case tho, the other file could just import static_filter_options instead of the function that returns it.

Reply:
Actually the function is preferred over the dict, I forgot to get rid of the dict. 

Would pref the function because the output of the`get_filter_static_options()` function can be modified (which the `get_all_filter_options()` function needs to do). If we were to just use the dict, we would need to make a copy of it in order to modify it.